server/lib/calendar/calendar.findCurrentlyRunningEvent.js

```diff
@@ -41,8 +41,9 @@ async function findCurrentlyRunningEvent(calendars, calendarEventNameComparator,
   };
   // eslint-disable-next-line default-case
   switch (calendarEventNameComparator) {
-    // do nothing in that case
+    // do nothing in that cases
     case 'has-any-name':
+    case 'does-not-contains':
```

Comment:
Still not sure why this function does the same thing for 'has-any-name' and 'does-not-contains' 

On peut parler en français si c'est plus simple pour toi pour expliquer :)